### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by Mhaiyang** [TuesdayT/CVPR2020-A2dele#2]
Hi, I am confused by the "temperature" parameter in the "cross_entropy2d" and "KD_KLDivLoss". What is the function of this parameter? Thanks!

**Comment by TuesdayT** [comment]
Hi！The temperature parameter is a hyper-parameter in knowledge distillation. It can be used in the classification task (saliency detection can be seen as a pixel-wise binary classification) to increase the difficulty of classification and improve the ability of mining difficult samples. In detail, please refer to https://nervanasystems.github.io/distiller/knowledge_distillation.html


On 07/23/2020 07:36, TaylorMei wrote:

Hi, I am confused by the "temperature" parameter in the "cross_entropy2d" and "KD_KLDivLoss". What is the function of this parameter? Thanks!
